WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -32,7 +32,11 @@
     <% end %>
 
     <% if !@persons.empty? %>
-      <h2><%= anchorable "People" %></h2>
+      <h2>
+        <%= anchorable "People" %>
+        |
```

Comment:
It separates *Persons* from the link to 'advanced search' that leads to the persons index with appropriate params.
![image](https://cloud.githubusercontent.com/assets/17034772/15628519/45772e14-2503-11e6-9245-0d485fd136e4.png)